syft/pkg/cataloger/internal/pkgtest/test_generic_parser.go

```diff
@@ -179,9 +180,29 @@ func (p *CatalogTester) assertPkgs(t *testing.T, pkgs []pkg.Package, relationshi
 		t.Errorf("unexpected packages from parsing (-expected +actual)\n%s", diff)
 	}
 
-	if diff := cmp.Diff(p.expectedRelationships, relationships, p.compareOptions...); diff != "" {
-		t.Errorf("unexpected relationships from parsing (-expected +actual)\n%s", diff)
+	if err := SubsetRelationship(p.expectedRelationships, relationships, p.compareOptions...); err != nil {
+		diff := cmp.Diff(p.expectedRelationships, relationships, p.compareOptions...)
+		t.Errorf("unexpected %s relationships from parsing (-expected +actual)\n%s", err, diff)
```

Comment:
I'm not entirely sure this is ideal -- if there is a diff, the diff will be very noisy and not represent what the check is actually doing. Could we just update the individual tests to have the right expected relationships?

Reply:
Np, do you want me to as simply 150 relationship to the static go code? Should we remove some from the sbom?

Reply:
Fyi the diff line your referring is simply to output for failed subset check above it was simpler for me to use it's full diff output for test outpu
But I understand you concern about the subset comparison

Reply:
I'd be good with updating the source SBOM to make a manageable set of relationships to add explicitly to the tests.